client/components/main/header/AccessibilityPane.jsx

```diff
@@ -14,6 +14,12 @@ const AccessibilityPane = () => {
     updateIsOpen(!isOpen);
   };
 
+  const handleKeyDown = e => {
+    if (e.key !== 'Tab' && e.key !== 'Shift' && e.key !== 'Alt') {
```

Comment:
Good call on this. It looks like the expected keys for activating buttons are Spacebar or Enter. Instead of activating on all keypresses except Tab/Shift/Alt, it's probably better to activate only on Spacebar or Enter.

https://webaim.org/techniques/keyboard/#testing